src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -269,7 +273,6 @@ private void decodeOtherParameter(Position position, int id, ByteBuf buf, int le
                 break;
             case 90:
                 position.set(Position.KEY_DOOR, readValue(buf, length, false));
-                break;
```

Comment:
This would simply break the decoder. Please make sure you test everything thoroughly before asking for final review.